config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java

```diff
@@ -204,6 +207,11 @@ public void addDefaultHandlersWithVip() {
         addVipHandler();
     }
 
+    private void addTestrunnerComponentsIfTester(DeployState deployState) {
+        if (deployState.getProperties().applicationId().instance().isTester())
+            addPlatformBundle(Paths.get(Defaults.getDefaults().underVespaHome("vespa-testrunner-components-jar-with-dependencies.jar")));
```

Comment:
Shouldn't this be 
```
Defaults.getDefaults().underVespaHome("lib/jars/vespa-testrunner-components-jar-with-dependencies.jar")
```
?

Reply:
Yes, it should. Fixed.

Reply:
Copied the old string to be _certain_ the path was correct, without typos ... but didn't think about the prefix being elsewhere :p